instance/management/commands/migrate_ephemeral.py

```diff
@@ -59,11 +59,14 @@ def handle(self, *args, **options):
         for ref in refs:
             # Disable ephemeral support and spawn a new AppServer
             LOG.info('Migrating %s to use persistent databases', ref.instance.name)
+            old_server_ids = list(ref.instance.appserver_set.filter(_is_active=True).values_list('id', flat=True))
```

Comment:
@Kelketek, how are the changes in this file relevant to this PR? Also @viadanna (who I remember working on this a long time ago), can we remove this management command and all the related code if we don't need it any more?

Reply:
I'm not familiar with why this management command exists, but I do know how my changes affected what the function calls actually do, so when I found this file, which referenced the functions I changed, I knew this code would crash if not updated, and I updated it.

Happy to remove the command altogether while I'm here if it's no longer needed, though.